src/constants/index.js

```diff
@@ -80,6 +80,11 @@ export const MARKER_AUTO_SELECT_MAX_DISTANCE = 0.75;
 
 export const TOTAL_ARCS = 20;
 
+/**
+ * Whether or not the the active location marker should be deselected if clicking outside the canvas
```

Comment:
A couple of super minor things:
1. Typo: `or not the the active` (two `the`)
2. Could be worded as `Determines if the active location marker should be deselected if clicking outside the canvas` 🐒

Reply:
Changed to what you said 👍